src/System.Runtime.InteropServices.RuntimeInformation/tests/CheckPlatformTests.netcoreapp.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace System.Runtime.InteropServices.RuntimeInformationTests
+{
+    public partial class CheckPlatformTests
+    {
+        [Fact, PlatformSpecific(TestPlatforms.FreeBSD)]
+        public void CheckFreeBSD()
+        {
+            Assert.True(RuntimeInformation.IsOSPlatform(OSPlatform.FreeBSD));
+            Assert.True(RuntimeInformation.IsOSPlatform(OSPlatform.Create("FREEBSD")));
+
+            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("DARWIN")));
+            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("LINUX")));
+            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("linux")));
+            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("NetBSD")));
+            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("ubuntu")));
+            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("UNIX")));
+            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Linux));
+            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.OSX));
+            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Windows));
+        }
+    }
```

Comment:
This will validate the behavior of FREEBSD when running on FreeBSD, but we don't have any tests validating the behavior of FREEBSD when running on something else.  So we should add a test like:
```C#
[Fact, PlatformSpecific(~TestPlatforms.FreeBSD))]
public void CheckFreeBSDOnOtherPlatforms()
{
    Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.FreeBSD));
    Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("FREEBSD"))); 
}
```

Reply:
ok. I can do that. Good chunk of that is in the standard test but proposed test would make t complete.